### PR TITLE
Bug 1878301: Improve display of node unschedulable status

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @TheRealJon
/cc @spadgett

### CODE REVIEW

File: frontend/packages/console-app/src/components/nodes/popovers/MarkAsSchedulablePopover.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { NodeKind } from '@console/internal/module/k8s';
+import NodeStatus from '../NodeStatus';
+import { makeNodeSchedulable } from '../../../k8s/requests/nodes';
+
+const MarkAsSchedulablePopover: React.FC<MarkAsSchedulablePopoverProps> = ({ node }) => {
+  const onClickMarkAsSchedulable = async () => {
+    try {
+      await makeNodeSchedulable(node);
+    } catch (err) {
+      console.error(err); // eslint-disable-line no-console
```
Comment (spadgett) [inline frontend/packages/console-app/src/components/nodes/popovers/MarkAsSchedulablePopover.tsx:14]: We want to surface this error to the user. Maybe use `errorModal` here?

File: frontend/packages/console-app/src/components/nodes/popovers/MarkAsSchedulablePopover.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const MarkAsSchedulablePopover: React.FC<MarkAsSchedulablePopoverProps> = ({ node }) => {
+  const [visible, setVisible] = React.useState<boolean>(null);
+
+  React.useEffect(() => {
+    setVisible(null);
+  }, [visible]);
+
+  const onClickMarkAsSchedulable = async () => {
+    try {
+      return await makeNodeSchedulable(node);
```
Comment (spadgett) [inline frontend/packages/console-app/src/components/nodes/popovers/MarkAsSchedulablePopover.tsx:19]: Is the `return` needed?

File: frontend/packages/console-app/src/components/nodes/popovers/MarkAsSchedulablePopover.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    } catch (err) {
+      setVisible(false);
+      return errorModal({ error: err.message || 'An error occurred. Please try again' });
```
Comment (spadgett) [inline frontend/packages/console-app/src/components/nodes/popovers/MarkAsSchedulablePopover.tsx:22]: Is the `return` needed?

**Review by spadgett — COMMENTED** [review]
Looks good, just a small question on whether we really need to return a value from the click handler.

**Review by cyril-ui-developer — COMMENTED** [review]
> Looks good, just a small question on whether we really need to return a value from the click handler.

@spadgett To make EsLint happy. `Expected to return a value at the end of async arrow function.eslint(consistent-return)`

**Comment by cyril-ui-developer** [comment]
> > Looks good, just a small question on whether we really need to return a value from the click handler.
> 
> @spadgett To make EsLint happy. `Expected to return a value at the end of async arrow function.eslint(consistent-return)`

@spadgett Please disregard the above comment. Removed  both `return`'s  and ran eslint again, and it is happy now.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**